academic-services/project-academic-knowledge/reference-interpret-method.md

```diff
@@ -40,53 +44,128 @@ Name | Description
 **interpretations[x].rules[y].name** | Name of the rule.
 **interpretations[x].rules[y].output** | Output of the rule.
 **interpretations[x].rules[y].output.type** | The data type of the output of the rule.  For the Academic Knowledge API, this will always be "query".
-**interpretations[x].rules[y].output.value** | he output of the rule. For the Academic Knowledge API, this is a query expression string that can be passed to the evaluate and calchistogram methods.
-**aborted** | True if the request timed out.
+**interpretations[x].rules[y].output.value** | The output of the rule. For the Academic Knowledge API, this is a query expression string that can be passed to the evaluate and calchistogram methods.
+**interpretations[x].rules[y].output.entities** | Optional array of top entities matching the interpretation.
+**timed_out_count** | The number of underlying index partitions that timed out when generating interpretations. 
```

Comment:
>The number of underlying index partitions [](start = 22, length = 41)

do we want to expose the underlying arch to users?  Not sure what they would do with this information...

Reply:
I was asking myself that question as I was writing this. I need to say something about it, as if I just ignore it is highly confusing what the heck it is.

Any suggestions for how I could make it more relatable?

---
In reply to: [395375353](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/130#discussion_r395375353) [](ancestors = 395375353)

Reply:
Hmm, we can say this is a parameter that is used internally to support the service.